onnxruntime/core/session/inference_session.cc

```diff
@@ -955,12 +1020,45 @@ class InferenceSession::Impl {
     return Status::OK();
   }
 
+  Status ValidateLevel(unsigned int level) const {
```

Comment:
>ValidateLevel [](start = 9, length = 13)

ValidateTransformerLevel would be a clearer name given this is in inference_session.cc. Should also be static. 

Or make it a lambda in SetTransformerContext given it's only used there isn't it

Reply:
Moving this and SetTransformerContext to transformer_utils... 

---
In reply to: [265377682](https://github.com/Microsoft/onnxruntime/pull/608#discussion_r265377682) [](ancestors = 265377682)